django_filters/filters.py

```diff
@@ -217,26 +217,26 @@ def is_noop(self, qs, value):
         return False
 
     def filter(self, qs, value):
-        value = value or ()  # Make sure we have an iterable
-
-        if self.is_noop(qs, value):
+        if not value:
```

Comment:
Duh - generally you need to perform an empty value check, because there are valid falsey values such as... `False` that you may want to filter on. In the case of the multiple choice filter, the incoming value should be an iterable anyway, so special handling shouldn't be necessary.